crates/gradbench/src/main.rs

```diff
@@ -675,6 +728,7 @@ fn cli() -> Result<(), ExitCode> {
                 }
             }
         }
+        Commands::Log { command } => log_command(command).map_err(|err| err_fail(anyhow!(err))),
```

Comment:
No need to wrap this in another `anyhow!`:

```suggestion
        Commands::Log { command } => log_command(command).map_err(err_fail),
```